server/live/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Disaster Ninja by kontur.io</title>
+  <meta name="description"
+        content="Disaster Ninja is a platform for assesment of mapping needs in case of natural disaster. See what's happening, what are available map sources, explore coverage quality of OpenStreetMap, chart the area of interest, export and import as geojson.">
+  <meta http-equiv="refresh" content="0; url=/active/"/>
```

Comment:
I suspect this way it may happen that it works faster than JS in some cases. changing 0 -> 1 should give enough time for it to work every time.